src/multi_select.coffee

```diff
@@ -22,6 +22,11 @@ Ember.Widgets.MultiSelectComponent = Ember.Widgets.SelectComponent.extend
   placeholder: undefined
   persistentPlaceholder: undefined
 
+  # disable tabindex of the component container to set focus directly to
+  # the input field, which is always visible. This helps reducing one tab
+  # step to navigate back to the previous component
+  tabindex: -1
```

Comment:
oh, ok - great! Since the other comment was still around, I figured you hadn't addressed it; you might want to comment "done" or something as a reply next time so I can easily tell the comment has been resolved when I skim through.